x-pack/plugins/infra/public/pages/home/index.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { HomePageContent } from './page_content';
+import { HomeToolbar } from './toolbar';
+
+import { EmptyPage } from '../../components/empty_page';
+import { Header } from '../../components/header';
+import { ColumnarPage } from '../../components/page';
+
+import { WithWaffleFilterUrlState } from '../../containers/waffle/with_waffle_filters';
+import { WithWaffleTimeUrlState } from '../../containers/waffle/with_waffle_time';
+import { WithKibanaChrome } from '../../containers/with_kibana_chrome';
+import { WithSource } from '../../containers/with_source';
+
+export class HomePage extends React.PureComponent {
+  public render() {
+    return (
+      <ColumnarPage>
+        <WithSource>
+          {({ metricIndicesExist }) =>
+            metricIndicesExist || metricIndicesExist === null ? (
+              <>
+                <WithWaffleTimeUrlState />
+                <WithWaffleFilterUrlState />
+                <Header />
+                <HomeToolbar />
+                <HomePageContent />
+              </>
+            ) : (
+              <WithKibanaChrome>
+                {({ basePath }) => (
+                  <EmptyPage
+                    title="Looks like you don't have any metrics indices."
+                    message="Let's add some!"
+                    actionLabel="Setup Instructions"
+                    actionUrl={`${basePath}/app/kibana#/home/tutorial_directory/metrics`}
```

Comment:
Instead of using `basePath` couldn't you just use `../`. That would eliminate the need to rely on `basePath` and it should work the same.

Reply:
What's wrong with `basePath`?

Reply:
Ok, I'm fine either way, just curious.

Reply:
we need to do that because we are linking to Page outside of our sap